Source/AzPester.psm1

```diff
@@ -56,7 +60,13 @@ function Invoke-AzPester {
             Set-AzContext -Context $contexts["default"].Context
 
             $container = New-PesterContainer -Path '*' -Data @{ Definition = $definitionWithParameters.definition }
-            Invoke-Pester -Container $container -Output 'Detailed'
+            
+            if ($Tags -contains "all") {
+                Invoke-Pester -Container $container -Output 'Detailed'
```

Comment:
I suppose you can only have one line Invoke-Pester -Container $container -TagFilter $Tags -ExcludeTagFilter $ExcludeTags -Output 'Detailed'. If args are null does it complain ?